specs/merge/beacon-chain.md

```diff
@@ -218,12 +218,9 @@ def is_execution_enabled(state: BeaconState, body: BeaconBlockBody) -> bool:
 
 #### `compute_timestamp_at_slot`
 
-*Note*: This function is unsafe with respect to overflows and underflows.
```

Comment:
Technically the overflow is not removed, but we do have a spec notice about everything being essentially safe-math, raising an error on overflow. Removing the comment is fine.